src/network/realtime.js

```diff
@@ -454,17 +453,23 @@ createNameSpace("realityEditor.network.realtime");
             frame: frameKey
         }));
 
+        // assuming that there is a single node per frame (thus skipping another level for nodeKey)
```

Comment:
This logic change appears to break avatar objects locally for me (the pointer and camera indicators). Have you tested them with this change in place?